### DISCUSSION TITLE
Set Fan Level

### DISCUSSION THREAD

**Discussion by delinend** [veista/nilan#89] [Q&A]
Hi.

I have just tryed to install nilan-dev branch on my HA, and it just work fine.

I have a CTS602 panel and using a RS485 RTU.

But I can't anywhere see how to control the fan speed/level (1, 2, 3, 4) in HA ? I got 43 "units" in my HA.

A'm I missing something ?

Btw: Firmware: 2.37.c
Comfort 300LR

Best regards

**Comment by veista** [comment]
It is best to install using HACS, just turn on the beta releases.

The fan setting is under the HVAC entity.

**Reply by delinend** [reply]
Thanks veista.

I found it :-)

Now I'm trying to use HA automations, but I can only set hvac_mode (off, auto, heat, cool). I can't select "Fan mode" (speed).

I want to change speed on a special time of the day or a push button.

Best regards

**Reply by veista** [reply]
#28

**Reply by Pjc78** [reply]
Same can not find a way to set fan speed

**Reply by Pjc78** [reply]
sorry for being an idiot, found it in the features. amazing
